src/java.desktop/macosx/native/libosxapp/NSApplicationAWT.m

```diff
@@ -159,17 +157,10 @@ - (void)finishLaunching
 
     [super finishLaunching];
 
-    [[NSUserNotificationCenter defaultUserNotificationCenter] setDelegate:self];
-
     // inform any interested parties that the AWT has arrived and is pumping
     [[NSNotificationCenter defaultCenter] postNotificationName:JNFRunLoopDidStartNotification object:self];
```

Comment:
looks good here, identical to jdk11u-dev patch.

Reply:
Thanks for checking!